docs/continuous_replication.md

```diff
@@ -43,19 +43,29 @@ To protect the replication, we removed the possibility to boot your copied VM di
 
 ### Job creation
 
-Create the Continuous Replication backup job, and leave it disabled for now. On the main Backup-NG page, note its identifiers, the main `backupJobId` and the ID of one on the schedules for the job, `backupScheduleId`.
+Create the Continuous Replication backup job, and leave it disabled for now. On the main Backup-NG page, copy the job's `backupJobId` by hovering to the left of the shortened ID and clicking the  copy to clipboard button:
```

Comment:
I'm not sure these “variable names” (`backupJobId`, `backupScheduleId`) are necessary.

What do you think?

Reply:
I think they help keep track of what order everything should go in for the final command, if we remove them I think we still need some way of naming each UUID so users can know where they go in the end command